guides/common/modules/proc_attaching-red-hat-subscriptions-to-content-hosts.adoc

```diff
@@ -5,8 +5,6 @@ Using activation keys is the main method to attach subscriptions to content host
 However, an activation key cannot update an existing host.
 If you need to attach new or additional subscriptions, such as future-dated subscriptions, to one host, use the following procedure.
```

Comment:
This entire file should be removed. You no longer can attach subscriptions of any kind to content hosts.

Reply:
This file still needs to be removed